src/tea_html_cmds.res

```diff
@@ -1,14 +1,19 @@
+
+open Webapi
+open Webapi.Dom
```

Comment:
Again, I think it would be better to skip the opens

Reply:
Changes are done.